### PR TITLE
Fix php setting mysqli charset

### PR DESCRIPTION
**Pull request by ChazyTheBest**
Not tested, should work.

### CODE REVIEW

**Comment by Aokromes** [comment]
no luck XD

**Comment by Aokromes** [comment]
@Star-lion any idea on this? XD

File: libs/db_lib/mysqli.php [hunk 1]
```diff
@@ -15,11 +15,13 @@ function connect($db_host, $db_username, $db_password, $db_name = NULL, $use_nam
 
         if (strpos($db_host, ':') !== false) list($db_host, $db_port) = explode(':', $db_host);
 
-        if (isset($db_port)) $this->link_id = @mysqli_connect($db_host, $db_username, $db_password, $db_name, $db_port);
-        else $this->link_id = @mysqli_connect($db_host, $db_username, $db_password, $db_name);
+        if (isset($db_port)) $this->link_id = new mysqli($db_host, $db_username, $db_password, $db_name, $db_port);
+        else $this->link_id = new mysqli($db_host, $db_username, $db_password, $db_name);
```
Comment (Gooyeth) [inline libs/db_lib/mysqli.php:19]: Maybe it's better: `@new mysqli` to avoid showing php errors

**Comment by Aokromes** [comment]
superseded by fe21019

**Pull request closed without merging**